requests/packages/idna/codec.py

```diff
@@ -0,0 +1,118 @@
+from idna.core import encode, decode, alabel, ulabel, IDNAError
```

Comment:
Ok, I've added the auto-rewrite to the `Makefile` and have rerun it to confirm that the imports are the only thing that get rewritten.